server/db/migrations/20200124175526-create-issues.js

```diff
@@ -34,11 +34,11 @@ module.exports = {
     },
     issuedDate: {
       allowNull: false,
-      type: Sequelize.DATE,
+      type: Sequelize.STRING,
     },
     returnDate: {
       allowNull: false,
-      type: Sequelize.DATE,
+      type: Sequelize.STRING,
```

Comment:
Why did you change the datatypes of issuedDate and returnDate to strings?

Reply:
if I would have kept to type: Sequelize.DATE, it would automatically add return date 
and I want that to be empty or even a string so that on the return that it can be updated 😞

Reply:
@blaise But I think that is not the best way to solve the problem, because you can not explain how a field called issuedDate have string as datatype.

Reply:
@AbdoulNuru  you point is understandable 
if there is a better way to do it and prevent Sequelize from out inputting the date for us even before the book is returned please let me know